### DISCUSSION TITLE
Policy Violations Not showing up

### DISCUSSION THREAD

**Discussion by tocaleb** [DependencyTrack/dependency-track#3374] [Q&A]
We've tried a few things but we can't see to get policy violations to show up.
Settings ->
Copy Left Policy:
-License group is Copy Left
-License group is Weak Copy Left

-No filters.

We waited 24 hours but as far as we can tell no updates have occurred.

A few questions:
- Are there specific logs that we can check to try and pinpoint the issue.
-Is there a way to rerun the policy check manually or trigger it on demand.... 

Notes:
-Tried uploading a new BOM but have not had any updates, yet.
-Using v4.8.2

Thank you

**Comment by EvgeniaPatsoni** [comment]
Hello,

Same behavior here. Using  v4.11.7.
Did you manage to find a solution?

**Comment by fupgang** [comment]
> -Is there a way to rerun the policy check manually or trigger it on demand....

From the discussion in https://github.com/DependencyTrack/dependency-track/issues/2146:

>   Re-evaluation would happen in any of these cases:
>
>    Upon next SBOM upload
>    Upon manual reanalysis of the project
 >   At least 1-2x a day during recurring analysis

Manual reanalysis refers to this:

![371398836-cc576987-305d-4ae6-a83c-5102b5b46e49](https://github.com/user-attachments/assets/1d783124-f0de-4aa2-a048-6094cf22191f)

This runs the vulnerability scan as well as a policy scan.
The policy scan logs informationals like:

- "Evaluating x component(s) against applicable policies"
- "Policy analysis complete"
